bin/gwdetchar-lasso-correlation

```diff
@@ -605,14 +625,27 @@ if args.band_pass:
 page.add(htmlio.parameter_table(content, start=start, end=end))
 page.add('<hr class="row-divider">')
 
+if args.band_pass:
+    page.h2('Primary shannel spectra', id_='spectra')
+    page.div(style_='display: flex;flex-wrap:nowrap;justify-content:space-around;')
```

Comment:
@macedo22, inline style definitions like this are generally discouraged in favor of separate cascading style sheets (CSS), and same goes for the empty divs that follow. However, I think you can forego this problem altogether and use the standard plot scaffolding, something like:

```python
    page.h2('Primary shannel spectra', id_='spectra')
    page.div(class_='scaffold well')
    page.div(class_='row')
    page.div(class_='col-md-6')
    spectra_img1 = htmlio.FancyPlot(spectrum_plot_zoomed_out)
    page.add(htmlio.fancybox_img(spectra_img1))
    page.div.close()  # col-md-6
    page.div(class_='col-md-6')
    spectra_img2 = htmlio.FancyPlot(spectrum_plot_zoomed_in)
    page.add(htmlio.fancybox_img(spectra_img2))
    page.div.close()  # col-md-6
    page.div.close()  # row
    page.div.close()  # scaffold well
```

This structure takes full advantage of features native to twitter-bootstrap, so you have a scaffold well (the panel object with a drop-shadow) containing a row with two responsive, equal-size columns. (Note, "responsive" here means they will be formatted well on just about all screens regardless of size.)